api/src/main/java/com/epam/pipeline/acl/cluster/ClusterApiService.java

```diff
@@ -112,4 +113,11 @@ public AllowedInstanceAndPriceTypes getAllowedInstanceAndPriceTypes(final Long t
     public List<NodeDisk> loadNodeDisks(final String name) {
         return nodeDiskManager.loadByNodeId(name);
     }
+
+
+    public InstanceDNSRecord changeInstanceDNSRecord(final Long regionId, final InstanceDNSRecord dnsRecord,
```

Comment:
I'd suggest to move dns method/methods to a new apiService and manager classes, as `ClusterApiService` mostly works with kubernetes cluster, except for instance offers (but is another topic to discuss).